marklogic-client-api/src/main/java/com/marklogic/client/impl/DocumentMetadataPatchBuilderImpl.java

```diff
@@ -752,7 +757,8 @@ public void write(JSONStringWriter serializer) {
     public void write(XMLOutputSerializer out) throws Exception {
       XMLStreamWriter serializer = out.getSerializer();
       writeStartReplace(out, "/rapi:metadata/rapi:quality", null);
-      serializer.writeCharacters(String.valueOf(quality));
+      if(String.valueOf(quality) != null)
+        serializer.writeCharacters(String.valueOf(quality));
```

Comment:
Given that quality is an int, I believe it is guaranteed not to be null, which makes the test unnecessary.